.github/workflows/build.yml

```diff
@@ -25,3 +25,18 @@ jobs:
           name: security-profiles-operator
           authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
       - run: make nix nix-arm64
+
+  image:
+    runs-on: ubuntu-latest
+    env:
+      USER: security-profiles-operator+github
+      IMAGE: quay.io/security-profiles-operator/build
+      TOKEN: ${{ secrets.QUAY_TOKEN }}
```

Comment:
The token is already set within the repository, so the master run should push a new image `latest`.